### PR TITLE
Update zhhk strings and fix color codes

### PR DESCRIPTION
**Pull request by lifehome**

### CODE REVIEW

File: src/main/resources/messages/messages_zhhk.yml [hunk 1]
```diff
@@ -44,68 +43,68 @@ antibot_auto_disabled: '&8[&6用戶系統&8] &3不正常連接數已減少，防
...
 old_email_invalid: '&8[&6用戶系統&8] &c你所填寫的舊電郵地址並不正確。'
 email_invalid: '&8[&6用戶系統&8] &c你所填寫的電郵地址並不正確。'
 email_added: '&8[&6用戶系統&8] &a已新增你的電郵地址。'
 email_confirm: '&8[&6用戶系統&8] &5請重覆輸入你的電郵地址。'
 email_changed: '&8[&6用戶系統&8] &a你的電郵地址已更改。'
 email_send: '&8[&6用戶系統&8] &a忘記密碼信件已寄出，請查收。'
-# TODO email_show: '&2Your current email address is: &f%email'
-# TODO incomplete_email_settings: 'Error: not all required settings are set for sending emails. Please contact an admin.'
+email_show: '&8[&6用戶系統&8] &2你所使用的電郵地址為：&f%email'
+incomplete_email_settings: '&8[&6用戶系統&8] &c電郵系統錯誤，請聯絡伺服器管理員。 &7(err: mailconfig_incomplete)'
```
Comment (ljacqu) [inline src/main/resources/messages/messages_zhhk.yml]: Is there a reasoning behind this? `&7(err: mailconfig_incomplete)`

**Comment by ljacqu** [comment]
Thanks! Leaving open in case you want to answer my comment above. Rest looks great.

File: src/main/resources/messages/messages_zhhk.yml (hunk 1 above)
Comment (lifehome) [reply src/main/resources/messages/messages_zhhk.yml]: It is intended as the translated string means "Mail server error, please contact server admins." which gives a friendly bummer to player, not to admins. Note that the debug message you quoted is greyed out and is left there for moderators or admins who cannot access the server console instantly, to promptly respond on the incident.

Comment (ljacqu) [reply src/main/resources/messages/messages_zhhk.yml]: Fine by me, thanks for the explanation :)

**Pull request merged**